lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -210,28 +205,30 @@ class CategoryRenderer {
    * Renders the group container for a group of audits. Individual audit elements can be added
    * directly to the returned element.
    * @param {!ReportRenderer.GroupJSON} group
+   * @param {{expandable: boolean}} opts
    * @return {!HTMLDetailsElement}
    */
-  _renderAuditGroup(group) {
-    const auditGroupElem = /** @type {!HTMLDetailsElement} */ (this._dom.createElement('details',
+  _renderAuditGroup(group, opts) {
+    const expandable = opts.expandable;
+    const auditGroupElem = /** @type {!HTMLDetailsElement} */ (
+          this._dom.createElement(expandable ? 'details' :'div',
           'lh-audit-group lh-expandable-details'));
-    const auditGroupHeader = this._dom.createElement('div',
+    const auditGroupSummary = this._dom.createChildOf(auditGroupElem, 'summary',
+          'lh-audit-group__summary lh-expandable-details__summary');
+    const auditGroupHeader = this._dom.createChildOf(auditGroupSummary, 'div',
           'lh-audit-group__header lh-expandable-details__header');
-    auditGroupHeader.textContent = group.title;
-
-    const auditGroupDescription = this._dom.createElement('div', 'lh-audit-group__description');
-    auditGroupDescription.appendChild(this._dom.convertMarkdownLinkSnippets(group.description));
+    this._dom.createChildOf(auditGroupSummary, 'div',
+      `lh-toggle-arrow  ${expandable ? '' : ' lh--toggle-arrow-unexpandable'}`, {
+        title: 'See audits',
+      });
 
-    const auditGroupSummary = this._dom.createElement('summary',
-          'lh-audit-group__summary lh-expandable-details__summary');
-    const auditGroupArrow = this._dom.createElement('div', 'lh-toggle-arrow', {
-      title: 'See audits',
-    });
-    auditGroupSummary.appendChild(auditGroupHeader);
-    auditGroupSummary.appendChild(auditGroupArrow);
+    if (group.description) {
```

Comment:
another option is to put these as two templates, one expandable and one unexpandable. `_renderAuditGroup` could then just query the one it needs and then add a `description` as below if needed